RecoLocalTracker/SiPixelRecHits/src/PixelCPEGeneric.cc

```diff
@@ -113,26 +113,26 @@ PixelCPEGeneric::PixelCPEGeneric(edm::ParameterSet const & conf,
     xerr_endcap_def_=0.0020;
     yerr_endcap_= {0.00210};
     yerr_endcap_def_=0.00075;
-  } else { // isUpgrade=true, funny case
-    xerr_barrel_ln_= {0.00114,0.00104,0.00214};
-    xerr_barrel_ln_def_=0.00425;
-    yerr_barrel_ln_= {0.00299,0.00203,0.0023,0.00237,0.00233,0.00243,0.00232,0.00259,0.00176};
-    yerr_barrel_ln_def_=0.00245;
-    xerr_endcap_= {0.00151,0.000813,0.00221};
-    xerr_endcap_def_=0.00218;
-    yerr_endcap_= {0.00261,0.00107,0.00264};
-    yerr_endcap_def_=0.00357;
+  } else { // isUpgrade=true    
+    xerr_barrel_ln_= {0.00025, 0.00030, 0.00035, 0.00035};
```

Comment:
@delaere 
[somewhat academic question]

nrow=1 population in the barrel is very low, according to your slides.
This, I suppose, is from Lorentz angle drift.
Is the barrel nrow=1 err of only 2.5 um due to effective selection bias?
Meaning that to get a residual width so small all nrow=1 pixels have to be on the edges.
is the pixel "hit" position for nrow=1 still in the middle of the pixel?
In this case we probably assign a systematically biased value, by about 2 sigma.
It probably makes more sense to keep 25/sqrt(12) in this case.

Do you have a plot fit for this case?